addons/web/static/src/core/debug/debug_menu.xml

```diff
@@ -12,7 +12,7 @@
                     <t t-if="!element.hide">
                         <t t-if="element.type == 'item'">
                             <DropdownItem payload="{ callback: element.callback }">
-                                <a t-if="element.href" t-att-href="element.href" t-esc="element.description"/>
+                                <a t-if="element.href" t-att-href="element.href" t-esc="element.description" t-on-click.prevent=""/>
```

Comment:
Same fix must be applied for the user menu: support and documentation links are opened twice too.